Tests/feaLib/builder_test.py

```diff
@@ -154,6 +156,7 @@ def check_fea2fea_file(self, name, base=None, parser=Parser):
                 sys.stderr.write(line+"\n")
             self.fail("Fea2Fea output is different from expected. "
                       "Generated:\n{}\n".format("\n".join(actual)))
+        parser.ignore_comments = temp
```

Comment:
a `try: ... finally: ...`, or a [context manager](https://docs.python.org/3.6/library/contextlib.html#contextlib.contextmanager) could be used here to temporarily patch the `parser` module and ensure it is restored to the original state, whether or not exceptions are raised within the block.